WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3565,17 +3567,18 @@ e_byzantium = {
 		capital = 560 # Cherson
 		
 		hellenic_pagan = 600
-		bosporan_faith = 200
+		bosporan_faith = 1000
 		
 		allow = {
-			hidden_tooltip = {
+			OR = {
+				religion = bosporan_faith
```

Comment:
The kingdom has no particular ties with the religion justifying a such condition or change in my view,